systemd-sysusers: add page (#10693)
tldr-pages/tldr: pages/linux/systemd-sysusers.md

# systemd-sysusers

> > Creates system users and groups.

Reviewer: ```suggestion
> Create system users and groups.
```

> > Create system users and groups.
> > More information: <https://www.freedesktop.org/software/systemd/man/systemd-sysusers.html>.

Reviewer 2: ```suggestion
> Create system users and groups.
> If the config file is not specified, files in the `sysusers.d` directories are used.
> More information: <https://www.freedesktop.org/software/systemd/man/systemd-sysusers.html>.
```

- Create users and groups from a specific configuration file:

> `systemd-sysusers {{config_pathpath}}`

Reviewer 2: ```suggestion
`systemd-sysusers {{path/to/file}}`
```

> - Simulate processing a configuration file without creating users and groups to preview the entries that would be generated:
> 
> `systemd-sysusers --dry-run {{config_pathpath}}`

Reviewer 2: ```suggestion
- Process configuration files and print what would be done without actually doing anything:

`systemd-sysusers --dry-run {{path/to/file}}`
```

> - Print the contents of the specified configuration file:
> 
> `systemd-sysusers --cat-config {{config_path}}`

Reviewer 2: This doesn't work, the path is ignored.
If you want to do this, you can just use `cat` instead.

```suggestion
- Print the contents of all config files (before each file, its name is printed as a comment):

`systemd-sysusers --cat-config`
```

# systemd-sysusers

> > creates system users and groups.

Reviewer: ```suggestion
> Create system users and groups.
```

> More information: <https://www.freedesktop.org/software/systemd/man/systemd-sysusers.html>.

> - Create Users and Groups from a Specific Configuration File:

Reviewer: ```suggestion
- Create users and groups from a specific configuration file:
```

`systemd-sysusers {{config_pathpath}}`

- Simulate processing a configuration file without creating users and groups to preview the entries that would be generated:

`systemd-sysusers --dry-run {{config_pathpath}}`

- Print the contents of the specified configuration file:

`systemd-sysusers --cat-config {{config_path}}`

> 

Reviewer: ```suggestion
```

Author: Do you want me to remove this command ?

Reviewer: No, the extra line after the command.

Generally, you only need 1 line for having an end of line, any other additional one as added in code as blank lines, our linter doesn't allow these extra blank lines at last that's why I made this suggestion to remove it.
